src/Core/Domain/Product/QueryResult/ProductPricesInformation.php

```diff
@@ -85,6 +86,11 @@ class ProductPricesInformation
      */
     private $unitPriceRatio;
 
+    /**
+     * @var PriorityList|null
```

Comment:
1/ Type name is very different of the variable name ... are you sure ?
2/ Don't remember in which version of PHP we are allowed to prefix the name of the variable with a "?" to specify is nullable.

Reply:
1. Classname of PriorityList doesn't have a prefix (`specificPrice`) because it is in SpecificPrice namespace. so I could have also named the var $specificPricePrioritiesList, but the `list` suffix didn't seem to bring much use in this case :thinking:

Reply:
2. google it :smile: (7.1 i guess https://www.php.net/releases/7_1_0.php)
(or if you talking about smth like this  `private ?PriorityList $specificPricePriorities` then its from 7.4)